### ISSUE TITLE
Moving phpMyAdmin config inside puppet

### ISSUE THREAD

**Issue by mnapoli** [puphpet/puphpet#120]
I used the online Puphpet generator to create a vagrant/puppet config and all is good.

But I'd also like to use the puppet config on its own to configure some servers (running `puppet --modulepath modules manifests/default.pp`).

The only problem I have is that line in Vagrantfile:

``` ruby
  config.vm.provision :shell, :inline => 'echo -e "mysql_root_password=pass
controluser_password=pass" > /etc/phpmyadmin.facts;'
```

I'm trying to move that operation inside puppet, but no success :/

Here is what I've tried so far:

``` ruby
file { '/etc/phpmyadmin.facts':
  content => "mysql_root_password=pass\ncontroluser_password=pass",
  ensure  => 'file',
}
class { 'phpmyadmin':
  require => [Class['mysql::server'], Class['mysql::config'], Class['php'], File['/etc/phpmyadmin.facts']],
}
```

But I get:

```
Failed to parse template phpmyadmin/config.db.php:
  Filepath: /opt/vagrant_ruby/lib/ruby/gems/1.8/gems/puppet-2.7.19/lib/puppet/parser/templatewrapper.rb
  Line: 65
  Detail: Could not find value for 'pma_controluser_password' at 12:/tmp/vagrant-puppet/modules-0/phpmyadmin/templates/config.db.php
 at /tmp/vagrant-puppet/modules-0/phpmyadmin/manifests/init.pp:51 on node precise64
```

In short, it seems that phpMyAdmin is installed before the `/etc/phpmyadmin.facts` file is written (when I log into the vagrant machine the file doesn't exist).
